Docs/RHCL-RELEASE-GUIDE.md

```diff
@@ -0,0 +1,873 @@
+   - Examples:
+     - [`rhcl-1-2-rhcl-operator-prod-rhel9.yaml`](https://gitlab.cee.redhat.com/releng/konflux-release-data/-/blob/main/config/stone-prd-rh01.pg1f.p1/product/ReleasePlanAdmission/api-management/rhcl-1-2-rhcl-operator-prod-rhel9.yaml) (production)
+     - [`rhcl-1-2-authorino-stage-rhel9.yaml`](https://gitlab.cee.redhat.com/releng/konflux-release-data/-/blob/main/config/stone-prd-rh01.pg1f.p1/product/ReleasePlanAdmission/api-management/rhcl-1-2-authorino-stage-rhel9.yaml) (staging)
+     - [`rhcl-file-based-catalog-stage.yaml`](https://gitlab.cee.redhat.com/releng/konflux-release-data/-/blob/main/config/stone-prd-rh01.pg1f.p1/product/ReleasePlanAdmission/api-management/rhcl-file-based-catalog-stage.yaml) (FBC staging)
+
+5. **Enterprise Contract Policies**
+   - Standard policies: [`releng/konflux-release-data/config/stone-prd-rh01.pg1f.p1/product/EnterpriseContractPolicy/`](https://gitlab.cee.redhat.com/releng/konflux-release-data/-/tree/main/config/stone-prd-rh01.pg1f.p1/product/EnterpriseContractPolicy)
+   - RHCL custom build policy: [`config/policy.yaml`](https://github.com/rh-api-management/rh-connectivity-link/blob/alpha/config/policy.yaml) in rh-api-management repo
+     - Used during build/test phase (developer workspace)
+     - Defines allowed registries, temporary exemptions for CVE/RPM checks
+   - RHCL release policies (referenced in RPA `spec.policy` field):
+     - [`registry-api-management-prod`](https://gitlab.cee.redhat.com/releng/konflux-release-data/-/blob/main/config/stone-prd-rh01.pg1f.p1/product/EnterpriseContractPolicy/registry-api-management-prod.yaml) (production component releases)
+     - [`fbc-standard`](https://gitlab.cee.redhat.com/releng/konflux-release-data/-/blob/main/config/common/product/EnterpriseContractPolicy/fbc-standard.yaml) (file-based catalog releases)
+
+6. **Constraints**
+   - Located in: [`releng/konflux-release-data/constraints/product/api-management.yaml`](https://gitlab.cee.redhat.com/releng/konflux-release-data/-/blob/main/constraints/product/api-management.yaml)
+   - Validates RPA values (registries, tags, policies)
+
+7. **Pyxis Delivery Repositories**
+   - All target repositories must be defined in Pyxis before release
+   - Managed via: `releng/pyxis-repo-configs` repository
+
+---
+
+## Step-by-Step: Releasing RHCL Components
+
+This section covers releasing operator images and bundles (not FBC).
+
+### Step 1: Verify Builds are Complete
+
+1. **Check PipelineRuns**
+   ```bash
+   oc project api-management-tenant
+   oc get pipelineruns --sort-by=.metadata.creationTimestamp | tail -20
+   ```
+
+2. **Find Recent Successful Builds**
+   - Look for PipelineRuns with `Succeeded` status
+   - Note the Snapshot name (format: `{application}-{hash}`)
+
+3. **Inspect Snapshot**
+   ```bash
+   oc get snapshot {snapshot-name} -o yaml
+   ```
+   - Verify all expected components are present
+   - Note the component image digests
+
```

Comment:
We should also add a step here to verify that the bundle references are pointing to the correct operand images, as this is something that is frequently incorrect when using konflux auto-generated snapshots

Case in point the limitador snapshot that was released while I was on PTO with incorrect reference to a non released limitador image

Reply:
Do you happen to have some handy commands for checking the bundle references? If not, it's all good, I'll figure them out.